src/qlever/commands/example_queries.py

```diff
@@ -212,89 +227,245 @@ def sparql_query_type(self, query: str) -> str:
             return "UNKNOWN"
 
     @staticmethod
-    def parse_queries_file(queries_file: str) -> dict[str, list[str, str]]:
+    def construct_get_queries_cmd(
+        queries_file: str, query_ids: str, query_regex: str, ui_config: str
+    ) -> str:
         """
-        Parse a YAML file and validate its structure.
+        Construct get_queries_cmd from queries_tsv file if present or use
+        example queries by using ui_config. Use query_ids and query_regex to
+        filter the queries
+        """
+        get_queries_cmd = (
+            f"cat {queries_file}"
+            if queries_file
+            else f"curl -sv https://qlever.cs.uni-freiburg.de/"
+            f"api/examples/{ui_config}"
+        )
+        sed_arg = query_ids.replace(",", "p;").replace("-", ",") + "p"
+        get_queries_cmd += f" | sed -n '{sed_arg}'"
+        if query_regex:
+            get_queries_cmd += f" | grep -Pi {shlex.quote(query_regex)}"
+        return get_queries_cmd
```

Comment:
We discussed that there should be an additional option `--example-queries´ (and the whole command should be renamed to `benchmark-queries`, so that the code looks them same no matter whether you parse from TSV or YAML (just the parsing is different). The user should then *either* specify `--example-queries` or one of the `queries-...`, and there should be an informative error message otherwise